scripts/vite-server-setup.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
```

Comment:
add vite plugin to generate temp certs for tls server (+update docs)

Reply:
✔️